jetson/mp9250_bridge/main.cpp

```diff
@@ -251,7 +251,7 @@ int main(void)
     char r_buf[1024];
     bzero(r_buf,1024);
 
-    fd = uart_open(fd,"/dev/ttyS1");/*串口号/dev/ttySn,USB口号/dev/ttyUSBn */ 
+    fd = uart_open(fd,"/dev/ttyUSB0");/*串口号/dev/ttySn,USB口号/dev/ttyUSBn */ 
```

Comment:
Since we are using the FTDI UART->USB for everything I think this is appropriate.